ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -849,3 +935,48 @@ extension ClientQueryViewController: UITableViewDragDelegate {
 	}
 
 }
+
+// MARK: - UIImagePickerControllerDelegate
+extension ClientQueryViewController: UIImagePickerControllerDelegate {
+
+	func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
+
+		var name: String = "test.jpg"
+		var url: URL = URL(string: "https://demo.owncloud.org")!
```

Comment:
Both`name` and `url` should probably be optionals…

Reply:
Changed.